packages/lit-dev-content/site/tutorials/content/first-component/07.md

```diff
@@ -0,0 +1,78 @@
+In this section you will dynamically change the color of the `word-viewer` when
+the play direction changes.
+
+The `classMap` directive is a Lit directive that can dynamically toggle a list
+of classes on an element. `classMap` takes an object as an argument in which
+each key is treated as a class name, and if the value is truthy, the class is
+added to the element.
+
+Add the `backwards` class to the `<pre>` element when `this.playDirection` is
+`-1`.
+
+{% switchable-sample %}
+
+```ts
+// word-viewer.ts
+
+  render() {
+    ...
+    return html`<pre
+      class="${classMap({ backwards: this.playDirection === -1 })}"
+      ...
+    >${word}</pre>`;
+  }
+
+```
+
+```js
+// word-viewer.js
+import {classMap} from 'lit/directives/class-map.js';
+...
+
+  render() {
+    ...
+    return html`<pre
+      class="${classMap({ backwards: this.playDirection === -1 })}"
+      ...
+    >${word}</pre>`;
+  }
+
+```
+
+{% endswitchable-sample %}
+
+Now when `this.playDirection === -1`, the `<pre>` element will gain the class
+`backwards`.
```

Comment:
Maybe another opportunity to callout that this can be checked via dev tool? My initial instinct was to flip the order to add the `.backwards` style first then use the `classMap` to see the result but doing it in this order gives that opportunity to utilize the dev tool to inspect the element.

Reply:
Great recommendation. Adding a line to encourage inspecting with dev tooling.